.github/ISSUE_TEMPLATE/ql--false-positive.md

```diff
@@ -0,0 +1,36 @@
+---
+name: False positive
+about: Tell us about an alert that shouldn't be reported
```

Comment:
```suggestion
about: Report code scanning (CodeQL) alerts that you think should not have been detected (not applicable, not exploitable, etc.)
```

Reply:
Shall we replace `code scanning (CodeQL) alerts` with `CodeQL alerts` ? The template is not for arbitrary code scanning alerts; in addition alerts found directly by the CodeQL CLI are also in scope.

Reply:
Yep, makes sense